osquery/tables/networking/windows/arp_cache.cpp

```diff
@@ -59,27 +56,33 @@ QueryData genIPv4ArpCache(QueryContext& context) {
     long interfaceIndex;
     if (iface.count("interface") > 0) {
       safeStrtol(iface.at("interface"), 10, interfaceIndex);
-      std::string macAddress = iface.at("mac");
-
+      auto macAddress = iface.count("mac") > 0 ? iface.at("mac") : "";
       mapOfInterfaces.insert(std::make_pair(interfaceIndex, macAddress));
```

Comment:
Does `mapOfInterfaces[interfaceIndex] = iface.count("mac") > 0 ? iface.at("mac") : "";` not work